src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Color;
+  color : Shape;
```

Comment:
```suggestion
  shape: Shape;
  color: Color;
```
It seems you have mixed up the types)